src/main/java/com/mcmiddleearth/guidebook/data/InfoArea.java

```diff
@@ -48,11 +50,13 @@
 public abstract class InfoArea {
 
     private static final int CHAT_LENGTH = 90;
+    private static final int NEAR_DISTANCE = 10;
+    private static final Duration COOLDOWN = Duration.ofMinutes(1);

```

Comment:
Let me know if you think a minute is too short/long

Reply:
I think it's good